### PR TITLE
Initial commit to new repo and update to 1.1.0

### PR DESCRIPTION
**Pull request by andrasguseo**
Moving the extension to the new repository with nice fixes.

Changelog:
= 1.1.0 2018-06-13 =
* Fixed styling of mini calendar widget when used in a sidebar
* Fixed the color of the present day date in the mini calendar widget
* Adjusted the width of the day/date box (and event info box) in the sidebar so the day doesn't wrap when using a narrow sidebar
* Updated extension template
* Added readme.txt
* Added license.txt

Screenshots:
Before:
https://cloudup.com/c1oANu3Y9MZ
After:
https://cloudup.com/c4FpnWpQH8L

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Divi Theme Compatibility
+ * Plugin URI:        https://theeventscalendar.com/extensions/elegant-themes-divi-theme-compatibility/
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-divi-theme-compatibility
```
Comment (barryhughes) [inline index.php:5]: :boom:

File: index.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+    class_exists('Tribe__Extension')
```
Comment (barryhughes) [inline index.php]: Whitespace (1/3): here and throughout the change, the initial indent should be one or more tabs (not spaces)

File: index.php [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    && !class_exists('Tribe__Extension__Example')
```
Comment (barryhughes) [inline index.php]: Whitespace (2/3): there should be a space after any usage of the bang operator (ie `! class_exists( '' )`)

Comment (barryhughes) [reply index.php]: Whitespace (3/3): here and throughout the change, we should have spaces after and before the opening/closing parentheses (ie `do_func( 'foo', $bar )`)

**Review by barryhughes — DISMISSED** [review]
This is great, thanks for porting it across to the new repo/structure. Though minor/no functional impact, it would be great if you could take a few mins to fix the formatting/whitespace usage ... otherwise this is good to go :+1:

**Comment by andrasguseo** [comment]
Fixed whitespaces and added a ` function_exists( 'et_custom_posts_per_page' ) )` check.

**Review by barryhughes — APPROVED** [review]

**Pull request merged**